tools/run_tests/run_performance_tests.py

```diff
@@ -502,7 +502,7 @@ def main():
 
   if not args.dry_run:
     if remote_hosts:
-      archive_repo(languages=[str(l) for l in languages])
+      archive_repo(languages=list(set([l.split('_with_argv', 1)[0] for l in [str(l) for l in languages]])))
```

Comment:
This is better than the previous version, but would still need some polishing. Please look at the other comment for ideas how to modify 'build_performance.sh'

Reply:
Done. Create build_performance_php.sh, and install php dependence/ build protobuf c extension inside it.

Reply:
As noted in build_performance.sh   you can revert these changes.

Reply:
Done.